### PR TITLE
Chore/issue 197/navigation update and UI cleanup

### PR DESCRIPTION
**Pull request by estchang**
# Issue #197

## Description

Several UI tweaks and improvements.
- Removed redundant links to recipient dashboard in header
- Redid organizations page to save space
Screenshot 2024-11-03 at 3 11 18 PM

- Added breadcrumbs to pages for convenience to return to previous pages
Screenshot 2024-11-03 at 3 12 05 PM

- Misc icon adds and UI refinements

## Change Type (delete non-relevant options)

- 🧹 Chore (maintenance tasks that don't add new features or fix bugs)

## Dependencies

- No dependencies

### CODE REVIEW

File: src/templates/partials/theader.html [hunk 1]
```diff
@@ -45,24 +38,33 @@
...
-                    <a class="navbar-item" href="{% url 'user_profile:profile' %}">Profile</a>
+                    <a class="navbar-item" href="{% url 'user_profile:profile' %}">
+                        <span class="icon is-small"><i class="fa fa-user" aria-hidden="true"></i></span>
+                        <span>Profile</span>
+                    </a>
                 {% endif %}
                 <div class="navbar-item">
                     {% if user.is_authenticated %}
-                        <a class="button is-outlined is-dark" href="{% url 'messaging:messaging' %}"><span>Messages</span></a>
+                        <a class="button is-outlined is-dark" href="#">
```
Comment (arsalananwar11) [inline src/templates/partials/theader.html:64]: The messaging URL (href) is wrong here. I think it got over writeen. Can you please update to the latest?

File: .gitignore [hunk 2]
```diff
@@ -34,4 +34,7 @@ staticfiles/
 .coverage
 
 # Qahtan's Notes
-qahtansnotes.txt
\ No newline at end of file
+qahtansnotes.txt
```
Comment (arsalananwar11) [inline .gitignore:37]: Can we remove member specific files from this to keep the code clean?

**Review by arsalananwar11 — CHANGES_REQUESTED** [review]
Just a few minor comments but overall, the UI features look amazing Estella, thanks for adding them.

File: .gitignore (hunk 2 above)
Comment (QahtanAlJammali) [reply .gitignore:37]: Sure, I had it in the .gitignore but can move elsewhere!

File: src/templates/partials/theader.html (hunk 1 above)
Comment (estchang) [reply src/templates/partials/theader.html]: Fixed! Thanks for catching

**Review by maxgrove42 — APPROVED** [review]
nice work!

**Review by QahtanAlJammali — APPROVED** [review]
LGTM! thanks for the great work!

**Review by arsalananwar11 — APPROVED** [review]
Looks great Estella, thanks!

**Pull request merged**